org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,95 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-22.08
```

Comment:
Why not use 5.15-23.08?

Reply:
I'm trying out 6.7 rn

Reply:
Ok, the baseapp should be available soonish. I created the branch but Flathub has having some server issues that delays publishing.

Reply:
6.6 if for qt6 but is giving error: